apps/web/src/app/admin-console/organizations/policies/policy-edit.component.ts

```diff
@@ -92,6 +105,12 @@ export class PolicyEditComponent implements AfterViewInit {
         throw e;
       }
     }
+    this.organization = await firstValueFrom(
+      this.organizationService.getOrganizationById$(this.data.organizationId),
+    );
+    this.isUpsellingEnabled = await this.organizationBillingService.isUpsellingPoliciesEnabled(
+      this.organization,
+    );
```

Comment:
following the comment on updating `isUpSellingPoliciesEnabled` to return an `Observable<boolean>`:
```suggestion
this.organization$ = this.accountService.activeAccount$.pipe(
      getUserId,
      switchMap(userId => this.organizationService.organizations$(userId)),
      getOrganizationById(this.data.organizationId)
    );

    this.isUpsellingEnabled$ = this.organization$.pipe(
      switchMap(organization => this.organizationBillingService.isUpsellingPoliciesEnabled$(organization))
    );

```